### ISSUE TITLE
Project still supported?

### ISSUE THREAD

**Issue by GarthHere** [Paradoxis/facebook-csharp-ads-sdk#6]
Hi there,

Really awesome effort and cant thank you enough for putting this project together.

Just before I go crazy developing against this code, is this project still supported?

Many thanks!
Garth

**Comment by Paradoxis** [comment]
@GarthHere Hey, thanks for reaching out! 😄 

**TL;DR:** Not really, I advise against using it for a large scale production-critical project, I'd reccomend using the [unofficial C# SDK for Facebook](https://github.com/facebook-csharp-sdk/facebook-csharp-sdk) and calling the `FacebookClient.get()` method like I do [here](https://github.com/Paradoxis/Facebook-CSharp-Ads-SDK/blob/master/src/FacebookAds/FacebookAds/Interfaces/AbstractCrudObject.cs#L200), or using the official [Python, Ruby, NodeJS, PHP, Java SDK's](https://github.com/facebook?utf8=%E2%9C%93&q=business) instead and wrapping C# code around it.

<hr>

**Full Aswer:** While I occasionally update the project when asked to do so (like in [this](https://github.com/Paradoxis/Facebook-CSharp-Ads-SDK/issues/2) instance), I no longer actively develop the project and advise against using it for large-scale production-critical projects, with the reasons being as follows:

* **I no longer write any projects in C#**, I mainly write code in Python these days as I work in cyber security (where it's mainly used) and I enjoy the flexibility / speed of development the language provides. 
* **I no longer work in anything marketing-related**, as previously mentioned I now work in cyber security. This project was written during an internship at [Searchresult B.V. ](https://www.searchresult.nl/) <sup>(awesome company, by the way)</sup> and I got permission to open source it. Since I now no longer work for any marketing based-companies I don't really have time to maintain it during work-hours.
* **I don't have a lot of free time anymore**, a more personal thing; but I just don't have much free time to spend on maintaining projects I don't use myself. I've got a full-time job and spend most of my weekends hanging out with friends as much as I can to prevent myself from being isolated from the real world haha. 😃

Further more, the project **only supports GET requests**, meaning you can't do a lot of things you probably would like to do. As (at the time) there wasn't an API playground for the ads SDK, and all calls to place ads, update ads, delete ads would result in charges being billed to your credit card (which, at the time wasn't the most useful thing as I was a broke college student), so it lacks **a lot** of useful features. 

Hope this clears things up 😄
